scipy-1.0/paper.tex

```diff
@@ -122,6 +122,21 @@ \section*{Project organisation and community}
 
 \textbf{Governance}
 
+SciPy adopted an official Governance Document on August 3, 2017 (see PR\#6953). In summary,
+we have a Benevolant Dictator for Life (BDFL) in Pauli Virtanen, and a (currently) 18
```

Comment:
The paper will be published in a pretty broad journal, so it might be best to mention what the role of the BDFL is.